main/dconf/APKBUILD

```
@@ -1,32 +1,28 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=dconf
-pkgver=0.26.0
+pkgver=0.32.0
 pkgrel=1
 pkgdesc="A low-level configuration system."
 url="http://live.gnome.org/dconf"
 arch="all"
 license="LGPL-2.1"
 depends=""
-makedepends="glib-dev dbus-dev docbook-xsl vala"
+makedepends="glib-dev dbus-dev docbook-xsl vala gtk+3.0-dev libxml2-dev meson py3-setuptools bash-completion"
 subpackages="$pkgname-dev $pkgname-doc"
 source="https://download.gnome.org/sources/dconf/${pkgver%.*}/dconf-$pkgver.tar.xz"
+builddir="$srcdir"/dconf-$pkgver
 
-#	0001-Avoid-posix_fallocate-so-it-works-on-uClibc.patch
-builddir="$srcdir"/$pkgname-$pkgver
-
-build () {
+build() {
 	cd "$builddir"
-	./configure \
-		--build=$CBUILD \
-		--host=$CHOST \
+	meson \
 		--prefix=/usr \
-		--libexecdir=/usr/lib/dconf
-	make V=1
+		. output
+	ninja -C output
 }
 
 package() {
 	cd "$builddir"
-	make DESTDIR="$pkgdir" install
+	DESTDIR="$pkgdir" ninja -C output install
 }
 
```

I had a closer look why you added bash-completion . So the pkg includes a bash-completion file, you should put it in a subpkg.

```suggestion

bashcomp() {
	pkgdesc="Bash completions for $pkgname"
	install_if="$pkgname=$pkgver-r$pkgrel bash-completion"

	mkdir -p "$subpkgdir"/usr/share
	mv "$pkgdir"/usr/share/bash-completion "$subpkgdir"/usr/share
}
```